### ISSUE TITLE
gtk_window_set_wmclass() deprecated in GTK 3.22

### ISSUE THREAD

**Issue by Ygrex** [aidanholm/luakit#153]
running make on clean c386400:
```
widgets/window.c: In function ‘widget_window’:
widgets/window.c:205:5: warning: ‘gtk_window_set_wmclass’ is deprecated [-Wdeprecated-declarations]
     gtk_window_set_wmclass(d->win, "luakit", "luakit");
     ^~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/gtk-3.0/gtk/gtkdialog.h:32:0,
                 from /usr/include/gtk-3.0/gtk/gtkaboutdialog.h:30,
                 from /usr/include/gtk-3.0/gtk/gtk.h:31,
                 from ./clib/widget.h:32,
                 from ./widgets/common.h:24,
                 from widgets/window.c:23:
/usr/include/gtk-3.0/gtk/gtkwindow.h:154:12: note: declared here
 void       gtk_window_set_wmclass              (GtkWindow           *window,
            ^~~~~~~~~~~~~~~~~~~~~~
```

**Comment by aidanholm** [comment]
The documentation seems to suggest that the mandated alternative is using GtkApplication, which luakit currently does not use. Switching to that might require fiddling with luakit's libunique system as well. I'll look further into it sometime, but currently there are more pressing bugs to fix... 152 is particularly irksome...

**Comment by aidanholm** [comment]
Fixed at db397f0; note the change of window class from `luakit` to `Luakit`.
